packages/js/src/utils/swc/compile-swc.ts

```diff
@@ -148,6 +148,9 @@ export async function* compileSwcWatch(
       };
 
       stderrOnData = (err?: any) => {
+        if(err.includes("Debugger attached.")) {
+          return;
+        }
         process.stderr.write(err);
```

Comment:
We should still log the Debugger attached message, so it should probably be reordered:

```suggestion
        process.stderr.write(err);
        if(err.includes("Debugger attached.")) {
          return;
        }
```